common/marshal/marshal.go

```diff
@@ -187,8 +187,12 @@ func MarshalJsonBlock(b *types.SerializedBlock, inclTx bool, fullTx bool,
 		{Key:"stateRoot", Val:head.StateRoot.String()},
 		{Key:"bits", Val:strconv.FormatUint(uint64(head.Difficulty), 16)},
 		{Key:"difficulty", Val:head.Difficulty},
-		{Key:"nonce", Val:head.Nonce},
-		{Key:"timestamp", Val:head.Timestamp.Format(time.RFC3339)},
+		{Key:"pow", Val : map[string]interface{}{
+			"nonce":head.Pow.GetNonce(),
+			"pow_type":pow.PowMapString[head.Pow.GetPowType()],
+			"proof_data":head.Pow.GetProofData(),
+		},},
+		{Key:"timestamp", Val:head.Timestamp.Format("2006-01-02 15:04:05.0000")},
```

Comment:
the time format is not correct. it overrides the previous fix #36

Reply:
fixed